### PR TITLE
Add cilium service mesh

### PR DESCRIPTION
**Pull request by HPrinz**

### CODE REVIEW

File: _includes/comparison.html [hunk 1]
```diff
@@ -23,6 +24,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>1.4</td>
         <td>1.7</td>
         <td>1.0</td>
+        <td>1.2</td>
```
Comment (tgraf) [inline _includes/comparison.html:27]: 1.12?

File: _includes/comparison.html [hunk 2]
```diff
@@ -63,6 +68,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>any</td>
         <td>any</td>
         <td><a href="https://release-v0-11.docs.openservicemesh.io/docs/guides/traffic_management/ingress/" target="_blank">prepared config for Contour</a>, compatible with any other</td>
+        <td>any</td>
```
Comment (tgraf) [inline _includes/comparison.html:71]: Cilium includes a full Ingress controller in addition to support any third party Ingress:
https://docs.cilium.io/en/v1.12/gettingstarted/servicemesh/ingress/

File: _includes/comparison.html [hunk 3]
```diff
@@ -93,6 +101,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
             <td></td>
             <td></td>
             <td></td>
+            <td></td>
```
Comment (tgraf) [inline _includes/comparison.html:104]: Yes
https://github.com/cilium/cilium/blob/master/USERS.md

File: _includes/comparison.html [hunk 4]
```diff
@@ -113,6 +123,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
             <td>Traefik Mesh currently does not support transparent TLS encryption.</td>
             <td>Kuma is possibly the most flexible service mesh. Teams should thoroughly consider whether their project can handle the complexity involved.</td>
             <td>OpenServiceMesh (OSM) is the latest service mesh Implementation and simply too young to be production-ready.</td>
+            <td>The different architecture of Cilium limits it's capabilities compared to ther products like TODO.</td>
```
Comment (tgraf) [inline _includes/comparison.html:126]: Cilium also uses Envoy in addition to eBPF so the capabilities are the same as with other Envoy-based service mesh.

File: _includes/comparison.html [hunk 5]
```diff
@@ -204,6 +221,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>Kubernetes</td>
         <td>Kubernetes, VMs, ECS</td>
         <td>Kubernetes</td>
+        <td>Kubernetes</td>
```
Comment (tgraf) [inline _includes/comparison.html:224]: Kubernetes, VMs, Metal

Comment (tgraf) [reply _includes/comparison.html:224]: External workload support (non-Kubernetes)
https://docs.cilium.io/en/v1.12/gettingstarted/external-workloads/#external-workloads

File: _includes/comparison.html [hunk 6]
```diff
@@ -214,6 +232,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td></td>
         <td></td>
         <td><a href="https://docs.microsoft.com/en-us/azure/aks/servicemesh-osm-about" target="_blank">Microsoft Azure</a></td>
+        <td></td>
```
Comment (tgraf) [inline _includes/comparison.html:235]: Cilium is directly integrated without requiring further installation into: EKS-A, GKE, Anthos, and DigitalOcean
All other cloud environments are supported via the default installation path

File: _includes/comparison.html [hunk 7]
```diff
@@ -224,6 +243,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>no</td>
         <td><a href="https://kuma.io/docs/latest/deployments/multi-zone" target="_blank">yes</a></td>
         <td>no</td>
+        <td>no</td>
```
Comment (tgraf) [inline _includes/comparison.html:246]: Yes
https://docs.cilium.io/en/v1.12/gettingstarted/external-workloads/#external-workloads

File: _includes/comparison.html [hunk 8]
```diff
@@ -251,6 +272,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://doc.traefik.io/traefik-mesh/configuration/#access-control" target="_blank">yes</a></td>
         <td>no</td>
         <td><a href="https://github.com/openservicemesh/osm#smi-specification-support" target="_blank">yes</a></td>
+        <td>no</td>
```
Comment (tgraf) [inline _includes/comparison.html:275]: roadmap

File: _includes/comparison.html [hunk 9]
```diff
@@ -368,6 +400,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://www.jaegertracing.io" target="_blank">Jaeger</a></td>
         <td><a href="https://www.jaegertracing.io" target="_blank">Jaeger</a>, <a href="https://zipkin.io/" target="_blank">zipkin</a>, <a href="https://kuma.io/docs/latest/policies/traffic-trace/#add-datadog-backend">datadog</a></td>
         <td><a href="https://www.jaegertracing.io" target="_blank">Jaeger</a></td>
+        <td></td>
```
Comment (tgraf) [inline _includes/comparison.html:403]: OpenTelemetry:
https://github.com/cilium/hubble-otel

File: _includes/comparison.html [hunk 10]
```diff
@@ -415,6 +451,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>no</td>
         <td><a href="https://kuma.io/docs/latest/policies/traffic-route/#l7-traffic-split" target="_blank">yes</a></td>
         <td>Header-based via <a href="https://github.com/servicemeshinterface/smi-spec/blob/master/apis/traffic-split/v1alpha3/traffic-split.md" target="_blank">SMI</a></td>
+        <td>no</td>
```
Comment (tgraf) [inline _includes/comparison.html:454]: Yes:
https://docs.cilium.io/en/v1.12/gettingstarted/servicemesh/envoy-traffic-management/

File: _includes/comparison.html [hunk 11]
```diff
@@ -432,6 +469,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://doc.traefik.io/traefik-mesh/configuration/#circuit-breaker" target="_blank">yes</a></td>
         <td><a href="https://kuma.io/docs/latest/policies/circuit-breaker/" target="_blank">yes</a></td>
         <td><a href="https://github.com/openservicemesh/osm/issues/980">yes</a></td>
+        <td>no</td>
```
Comment (tgraf) [inline _includes/comparison.html:472]: Yes:
https://docs.cilium.io/en/v1.12/gettingstarted/servicemesh/envoy-traffic-management/

File: _includes/comparison.html [hunk 12]
```diff
@@ -442,6 +480,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://doc.traefik.io/traefik-mesh/configuration/#retry" target="_blank">yes</a></td>
         <td>yes, <a href="https://kuma.io/docs/latest/policies/retry/" target="_blank">retry</a> and <a href="https://kuma.io/docs/1.1.0/policies/timeout" target="_blank">timeout</a></td>
         <td>no</td>
+        <td>no</td>
```
Comment (tgraf) [inline _includes/comparison.html:483]: Yes:
https://docs.cilium.io/en/v1.12/gettingstarted/servicemesh/envoy-traffic-management/

File: _includes/comparison.html (hunk 5 above)
Comment (HPrinz) [reply _includes/comparison.html:224]: I can only find Installation in Kubernetes in the docs. The external workload support is covered by the "Mesh Expansion" feature

File: _includes/comparison.html (hunk 6 above)
Comment (HPrinz) [reply _includes/comparison.html:235]: This section is about integrations on the cloud provider sides that provide config checksm automatic updates, etc.

**Comment by HPrinz** [comment]
Thanks a lot @tgraf , I have updated the PR 👍

**Pull request merged**